pages/windows/cd.md

```diff
@@ -1,20 +1,20 @@
 # cd
 
-> Displays the name of or changes the current working directory.
+> Display the path of or change the current working directory.
 > More information: <https://docs.microsoft.com/windows-server/administration/windows-commands/cd>.
 
-- Go to a directory in the same drive:
-
-`cd {{path/to/directory}}`
-
-- Display the name of the current directory:
+- Display the path of the current directory:
 
 `cd`
 
 - Go up to the parent of the current directory:
 
 `cd ..`
 
-- Go to a directory in a different drive:
+- Go to a specific directory in the same drive:
+
+`cd {{path\to\directory}}`
+
+- Go to a specific directory in a different [d]rive:
 
-`cd {{path/to/directory}} /d`
+`cd /d {{C}}:{{path\to\directory}}`
```

Comment:
Should this be C or 
```suggestion
`cd /d {{drive}}:{{path\to\directory}}`
```

Reply:
There is `/d` option which has a mnemonic. That's why I provided the sample value `C` instead of placeholder with description inside it.